File: src/BukkitPE/level/generator/normal/biome/GrassyBiome.php
<?php
namespace BukkitPE\level\generator\normal\biome;

use BukkitPE\block\Block;

abstract class GrassyBiome extends NormalBiome{

	public function __construct(){
		$this->setGroundCover([
			Block::get(Block::GRASS, 0),
			Block::get(Block::DIRT, 0),
			Block::get(Block::DIRT, 0),
			Block::get(Block::DIRT, 0),
			Block::get(Block::DIRT, 0),
		]);
	}
}

File: src/net/BukkitPE/level/generator/biome/GrassyBiome.java
package net.BukkitPE.level.generator.biome;

import net.BukkitPE.block.Block;
import net.BukkitPE.block.BlockDirt;
import net.BukkitPE.block.BlockGrass;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public abstract class GrassyBiome extends NormalBiome {
    public GrassyBiome() {
        this.setGroundCover(new Block[]{
                new BlockGrass(),
                new BlockDirt(),
                new BlockDirt(),
                new BlockDirt(),
                new BlockDirt()
        });
    }
}
